sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ColumnarBatch.java

```diff
@@ -30,7 +29,6 @@
  * batch so that Spark can access the data row by row. Instance of it is meant to be reused during
  * the entire data loading process.
  */
-@Evolving
```

Comment:
with the new changes to Columnar introduced in 3.0 I wonder if we shouldn't leave this evolving.

Reply:
I'll revert the changes to Columnar*